TestBookmarksRepairRequestor.swift

```diff
@@ -83,10 +98,18 @@ class TestBookmarksRepairRequestor: XCTestCase {
         let remoteClients = MockRemoteClientsAndTabs([ClientAndTabs(client: localClient, tabs: []), ClientAndTabs(client: remoteClient, tabs: [])])
         let validationInfo = [BufferInconsistency.missingValues: ["mock-guid1", "mock-guid2"]]
 
+        // Mock telemetry events
+        let mockFlowID = Bytes.generateGUID()
+        let startedEvent = makeRepairEvent(["started", nil, ["flowID": mockFlowID, "numIDs": "2"]])
+        let uploadEvent = makeRepairEvent(["request", "upload", ["flowID": mockFlowID, "deviceID": "IMPLEMENT ME", "numIDs": "2"]])
+        let abandonEvent = makeRepairEvent(["abandon", "silent", ["flowID": mockFlowID, "deviceID": "IMPLEMENT ME"]])
```

Comment:
Nitpick: if we could replace my "IMPLEMENT ME" deviceID by something less alarming :)